src/pepr/operator/controllers/istio/ambient-waypoint.ts

```diff
@@ -292,11 +290,10 @@ export async function reconcilePod(pod: a.Pod): Promise<void> {
     [ISTIO_WAYPOINT_LABEL]: waypointName,
   };
 
-  log.info(`Added waypoint labels to pod ${pod.metadata?.name}`, {
-    namespace,
-    waypointName,
-    clientId: matchingSso.clientId,
-  });
+  log.info(
+    { namespace, waypointName, clientId: matchingSso.clientId },
+    "Added waypoint labels to pod",
```

Comment:
Similar question to above, can we keep `${pod.metadata?.name}` in the log text?

Reply:
good catch!